examples/hello-world/hello-pt/requirements.txt

```diff
@@ -1 +1,3 @@
 nvflare[PT]>=2.3.0
+torch
+torchvision
```

Comment:
the first line already included torch and torchvision
Or should we NOT provide these options ([PT], [HE] etc)?

Reply:
either way works, personally I would do without [PT], but others may prefer the other way

Reply:
QA scripts also require installing a specific nvflare version.  It could not install RC packages with >=.  Therefore, that nvflare[PT]>=2.3.0 always installs 2.3.8, which is not what qa scripts requires.  We have to remove that line explicitly from requirements.txt and install torch torchvision separately.
I pushed another commit to remove torch torchvision and to install them directly from scripts.